js/views/settingsVw.js

```diff
@@ -91,6 +114,15 @@ module.exports = Backbone.View.extend({
     this.moderatorCount = 0;
 
     this.fetchModel();
+    
+    $(window).off('keydown.view').bind('keydown.view', function(e) {
```

Comment:
Please be sure to unbind this event in this view's remove function. Any events bound to an element outside of the view need to be unbound in the view's remove, otherwise it will leave the view as a ghost view, preventing it from being garbage collected.